cmd/ztest/ztest.c

```diff
@@ -7487,6 +7536,14 @@ main(int argc, char **argv)
 	 */
 	zfs_reconstruct_indirect_damage_fraction = 100;
 
+	/*
+	 * By design ztest will never inject damage in to the pool which is
+	 * beyond repair.  Allow the split block reconstruction code to
+	 * attempt all possible combinations.  This is required to prevent
+	 * ztest_scrub_impl() from misdiagnosing a persistent error.
+	 */
+	zfs_reconstruct_indirect_combinations_max = INT_MAX;
```

Comment:
It is possible, but so far I haven't seen that in my testing.  My concern is that if we don't set this then the scrub verification (spa error log size) might report false positive for reconstruction failures which exhaust the allowed attempts.  Let me comment this out for now if see how it goes, this may not be needed.